app/models/bet_option.rb

```diff
@@ -4,4 +4,8 @@ class BetOption < ApplicationRecord
 
   scope :winners, -> { where(winner: true) }
   scope :losers, -> { where(winner: false) }
+
+  def option_total
+    user_bets.map(&:amount_bet).sum
```

Comment:
Doesnt need to change but just pointing out that this calculation is happening in the application layer.

If you wanted it to happen at the database layer your could use `user_bets.pluck(:amount_bet).sum` I super recommend looking up the `pluck` docs, it can be a super useful tool to use.

Performing the calc in either the app layer or database layer has advantages and disadvantages. I dont really know what is most important here so there is no need to change it. (but maybe try and think what the advantages and disadvantages could be)